.github/workflows/test-dbt-installation-verify-test-jobs-statuses.yml

```diff
@@ -0,0 +1,86 @@
+# **what?**
+
+# **why?**
+
+# **when?**
+
```

Comment:
These need to be filled out.

Reply:
Fixed in https://github.com/dbt-labs/actions/pull/69/commits/bb27adf254e859b24ddc09319f419f0f599a8b74